resources/external_control.urscript

```diff
@@ -760,8 +782,10 @@ thread script_commands():
           friction_compensation_enabled = True
         end
       elif command == SET_PD_CONTROLLER_GAINS:
+        {% if ROBOT_SOFTWARE_VERSION > v5.10.0 %}
```

Comment:
I am wondering whether we should add some sort of warning if this is called on a software version where it is not used as calling it wont have any affect, the same goes for friction compensation and max joint torques? This warning could be in the driver or the script command object.